src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -161,7 +161,7 @@ void submit(FlightContext flightContext) throws DatabaseOperationException {
     }
 
     /**
-     * Record the flight state right before a step
+     * Record the flight state right after a step
```

Comment:
Is this just a comment correction, or are we changing how the steps are recorded because of the new shut down options and I'm just missing where we are doing that?